src/main/java/slimeknights/tconstruct/items/ToolItems.java

```diff
@@ -10,6 +10,9 @@
 import slimeknights.tconstruct.TConstruct;
 import slimeknights.tconstruct.common.registry.BaseRegistryAdapter;
 import slimeknights.tconstruct.library.TinkerRegistry;
+import slimeknights.tconstruct.library.tinkering.MaterialItem;
+import slimeknights.tconstruct.library.tools.*;
+import slimeknights.tconstruct.tools.*;
```

Comment:
Not a fan of wildcard imports, makes code difficult to trace without an IDE if you have multiple wildcards

Reply:
Yeah already talked to alex about it, seems to be a leftover before we enforced it in editorconfig. We want to cleanup all imports afterwards.